cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -47,10 +47,19 @@ extern template class raft::bench::ann::RaftCagra<uint8_t, uint32_t>;
 extern template class raft::bench::ann::RaftCagra<int8_t, uint32_t>;
 #endif
 #define JSON_DIAGNOSTICS 1
+#include "fix_latency_workload.h"
 #include <nlohmann/json.hpp>
 
 namespace raft::bench::ann {
 
+inline void parse_search_param(const nlohmann::json& conf,
+                               typename raft::bench::ann::FixLatencyWorkload::SearchParam& param)
+{
+  if (conf.contains("use_gpu")) param.use_gpu = conf.at("use_gpu");
+  if (conf.contains("sync_stream")) param.sync_stream = conf.at("sync_stream");
```

Comment:
Are these also renmants of the above `fix_latency_workload`? We aren't planning on adding new params for the user to specify, are we?